internal/web/commands.go

```diff
@@ -0,0 +1,76 @@
+package web
+
+import (
+	"fmt"
+	"io"
+	"os/exec"
+)
+
+func buildRawDisk(containerImage, outputDir string) string {
+	return fmt.Sprintf(`auroraboot \
+	--debug  \
+	--set "disable_http_server=true" \
+	--set "disable_netboot=true" \
+	--set "container_image=%s" \
+	--set "state_dir=%s" \
+	--set "disk.raw=true" \
+	`, containerImage, outputDir)
+}
+
+func buildISO(containerImage, outputDir, artifactName string) string {
+	return fmt.Sprintf(`auroraboot --debug build-iso \
+	--output %s \
+	--name %s \
+	docker:%s`, outputDir, artifactName, containerImage)
+}
+
+func buildOCI(
+	contextDir,
+	image string,
+	karosInitVersion string,
+	baseImage string,
+	variant string,
+	model string,
+	trustedBoot bool,
+	kubernetesProvider string,
+	kubernetesVersion string,
+) string {
+	return fmt.Sprintf(`docker build %s \
```

Comment:
Maybe this is for the future, but cant we just hook into the containerd libs here and create the image in pure go directly?